docs/en/enterprise-edition/content-collections/dashboards/create-and-manage-dashboards.adoc

```diff
@@ -37,7 +37,7 @@ If the filters in the Edit Dashboard workflow conflict with the filters selected
 |*Alerts by MTTR*
 |Displays alerts by severity and their mean time to resolution. 
 
-Note: If an alert is opened and resolved more than once a day, the latest resolution time is used for the MTTR calculation. A particular alert will be counted multiple times, if the same alert is resolved several times over multiple days.
+Note: If an alert is opened and resolved more than once a day, the latest resolution time is used for the MTTR calculation. An individual alert can be counted multiple times, if the same alert is resolved several times over multiple days. 
```

Comment:
try using the tt:[NOTE] tag or "*NOTE*" to differentiate it from the description.

Reply:
Done